src/module/module.py

```diff
@@ -11,7 +11,7 @@
 from smbus2 import smbus2 as smbus
 
 MILL_PER_SEC = 1000
-I2Cbus = smbus.SMBus(int(getenv('I2C_INTERFACE_NUMBER')))
+I2Cbus = smbus.SMBus(int(getenv('I2C_INTERFACE')[-1]))
```

Comment:
what if the interface number has more than one digit?

Reply:
good question, but the problem is SMBus.SMBus(int) takes an int as a parameter and I have to confirm full all the linux i2c interface editions, the number is put on the end .

Reply:
no, I'm talking about one digit vs several digits. currently you are only extracting one digit

Reply:
so weird to find any controller that has more than 10 (0 to 9) if that exists i2c interfaces many slaves possible for one interface).
I am with 1 digit.

Reply:
please make it also accept multiple digits to make it more future proof

Reply:
o propose to use both the i2c interface name and i2c number , maybe there is an interface named /dev/*-i2c ?